### PR TITLE
MOSIP-13160 : fixed default priority

### PR DESCRIPTION
**Pull request by MonobikashDas**

### CODE REVIEW

File: commons-packet/commons-packet-service/src/main/java/io/mosip/commons/packetmanager/service/PacketReaderService.java [hunk 1]
```diff
@@ -161,6 +163,60 @@ public String getSource(String id, String source, String process) {
...
+        SourceProcessDto sourceProcessDto = null;
+        if (StringUtils.isEmpty(source)) {
+            try {
+                if (defaultStrategy.equalsIgnoreCase(DefaultStrategy.DEFAULT_PRIORITY.getValue())) {
+                    InfoResponseDto infoResponseDto = info(id);
+                    ContainerInfoDto containerInfoDto = findPriority(field, process, infoResponseDto);
+                    if (containerInfoDto == null)
+                        throw new SourceNotPresentException();
+                    sourceProcessDto = new SourceProcessDto(containerInfoDto.getSource(), containerInfoDto.getProcess());
+                }
```
Comment (vishwa-vyom) [inline commons-packet/commons-packet-service/src/main/java/io/mosip/commons/packetmanager/service/PacketReaderService.java:176]: Not required to handle else for this if condition ?

File: pom.xml [hunk 2]
```diff
@@ -168,7 +168,7 @@
           </plugins>
         </build>
         <modules>
-           <module>kernel</module>
+           <!--<module>kernel</module>-->
```
Comment (vishwa-vyom) [inline pom.xml:171]: Why are we commenting the kernel module ?

Comment (MonobikashDas) [reply pom.xml:171]: we dont have any change in this patch(1.1.5.3, we have changes only in packetmanager

File: commons-packet/commons-packet-service/src/main/java/io/mosip/commons/packetmanager/service/PacketReaderService.java (hunk 1 above)
Comment (MonobikashDas) [reply commons-packet/commons-packet-service/src/main/java/io/mosip/commons/packetmanager/service/PacketReaderService.java:176]: it supports only default priority right now, there are 2 more priorities discussed which were not implemented yet, so when we implement other priority strategy then else block will come. Right now always DEFAULT_PRIORITY by default in our config.

Comment (vishwa-vyom) [reply commons-packet/commons-packet-service/src/main/java/io/mosip/commons/packetmanager/service/PacketReaderService.java:176]: we should throw a exception if there any spell mistake in the config, but for now it is ok

**Review by gsasikumar — APPROVED** [review]

**Pull request merged**